libsolutil/Whiskers.h

```diff
@@ -93,6 +93,7 @@ class Whiskers
 	Whiskers& operator()(std::string _parameter, long long);
 	void checkParameterValid(std::string const& _parameter) const;
 	void checkParameterUnknown(std::string const& _parameter) const;
+	void checkTemplateContainsTags(std::string const& _parameter, std::vector<std::string> const& _tags);
```

Comment:
Can you explain what parameter `_tags` is for?

Reply:
I added a comment.